rest/serverless_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 // Tests behaviour of CBG-2257 to poll only buckets in BucketCredentials that don't currently have a database
 func TestServerlessPollBuckets(t *testing.T) {
+	base.LongRunningTest(t)
```

Comment:
I'm not sure why these need to be added (and in one case below it's being added redundantly).  What aspect of these tests are inherently long-running?

Reply:
Originally, I added this when each of the serverless tests where taking over 10 seconds, however now seem to be faster locally. 

Don't know what's caused it to be faster (whether it was a laptop issue or the conversion to using the rest tester for all the tests or general test framework improvements) however this is great as the tests are much faster now.